service/controller/clusterapi/v26/resource/tccp/create.go

```diff
@@ -10,12 +10,12 @@ import (
 	"github.com/giantswarm/microerror"
 
 	"github.com/giantswarm/aws-operator/pkg/awstags"
-	"github.com/giantswarm/aws-operator/service/controller/legacy/v22/ebs"
```

Comment:
I fixed this as well. Will provide another PR for the legacy imports because the same issue is there as well.